mantle/platform/api/aws/images.go

```diff
@@ -329,21 +329,28 @@ func (a *API) CreateImportRole(bucket string) error {
 	return nil
 }
 
-func (a *API) CreateHVMImage(snapshotID string, diskSizeGiB uint, name string, description string, architecture string) (string, error) {
+func (a *API) CreateHVMImage(snapshotID string, diskSizeGiB uint, name string, description string, architecture string, volumetype string, imdsv2Only bool, uploadX86BootMode string) (string, error) {
```

Comment:
I guess here we can name that parameter something less wordy?

```suggestion
func (a *API) CreateHVMImage(snapshotID string, diskSizeGiB uint, name string, description string, architecture string, volumetype string, imdsv2Only bool, x86BootMode string) (string, error) {
```